content/release-10-17-0/whatsnew-10-17-0.md

```diff
@@ -52,3 +52,15 @@ A new option for dashboard configuration has been added that allows users to res
 ![Dashboard availability](/images/release-notes/dashboard-availability.png)
 
 For details, see [Cockpit > Working with dashboards](https://cumulocity.com/guides/10.17.0/users-guide/cockpit/#dashboards) in the *User guide*.
+
+#### Home Page Revision
+
+Improved the DM Homepage by replacing the map widget with Quicklinks and Recent Alarms. Also improved More Action Menu auto detects the number of items in it and display it as a button if it has only one menu item. 
+
+![Home Page Revision](/images/release-notes/home-page-revision.png)
+
+#### Confirmation pop-up
```

Comment:
@dmajsag I´m not sure if we should really add this here. I think it is sufficient to mention this in the improvements section. What do you think @l2p- ?